admin/services/data-access/check-form.data.service.js

```diff
@@ -34,38 +63,197 @@ const checkFormDataService = {
    * @param query
    * @param sortField
    * @param sortDirection
+   * @deprecated use sqlFetchSortedActiveFormsByName
    * @returns {Promise.<void>}
    */
   fetchSortedActiveForms: async (query, sortField, sortDirection) => {
     let sort = {}
     let q = query
-
+    winston.warn('check-form.data.service.fetchSortedActiveForms is deprecated')
     if (sortField && sortDirection) {
       sort[sortField] = sortDirection
     }
     q.isDeleted = false
     return CheckForm.find(q).sort(sort).lean().exec()
   },
 
+  /**
+   * Fetch active forms (not deleted)
+   * sorted by name
+   * @param windowId only forms assigned to the specified window (optional)
+   * @param sortDescending if true, sorts descending
+   * @returns {Promise<*>}
+   */
+  sqlFetchSortedActiveFormsByName: (windowId, sortDescending) => {
+    let sortOrder = 'ASC'
+    if (sortDescending) {
+      sortOrder = 'DESC'
+    }
+    const params = []
+    let sql = ''
+    if (windowId) {
+      sql = `SELECT * FROM [mtc_admin].[checkForm] WHERE isDeleted=0 
```

Comment:
I'm thinking that we should not hard-code the database name `[mtc_admin]`.  In my latest PR (soon forthcoming) I export this from sqlService.  We can leave as-is for now though.

Reply:
@jon-shipley this is the schema name, which is important

Reply:
however, given that the user only has access to this schema it may be irrelevant.  We should test this.